src/json_utils.h

```diff
@@ -4,13 +4,21 @@
 #if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
 
 #include <iomanip>
-#include <ostream>
 #include <limits>
+#include <ostream>
 #include <string>
+#include <string_view>
 
 namespace node {
 
-std::string EscapeJsonChars(const std::string& str);
+constexpr bool NeedsJsonEscape(std::string_view str) {
+  for (const char c : str) {
+    if (c == '\\' || c == '"' || c < 0x20) return true;
```

Comment:
nit: std::find_first_not_of might improve this function and reduce branch count

Reply:
Should compile to pretty much the same thing: https://godbolt.org/z/Mj8PY5vqK

Similarly with `std::none_of` and friends. Happy to change this, no strong feelings about it, but it's unlikely to actually result in different performance, I would say.

Reply:
Good to know. Thanks!